layers/+lang/lean/packages.el

```diff
@@ -0,0 +1,39 @@
+;;; packages.el --- Lean Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2014 Sylvain Benner
+;; Copyright (c) 2014-2015 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+(setq lean-packages '(mmm-mode f dash-functional
+                      (lean-mode :location (recipe
```

Comment:
@nixmaniack, thanks for your help so far. I'll make the changes , but since this is my first PR, I hope I'll be given the liberty to use multiple commits, I tried to rebase changes like this before and I ended up having to restart.